src/freertos_drivers/esp32/Esp32HardwareCanAdapter.hxx

```diff
@@ -94,10 +96,13 @@ public:
         ESP_ERROR_CHECK(can_driver_install(
             &can_general_config, &can_timing_config, &can_filter_config));
 
-        xTaskCreatePinnedToCore(rx_task, "ESP32-CAN RX", OPENMRN_STACK_SIZE,
-            this, RX_TASK_PRIORITY, &rxTaskHandle_, tskNO_AFFINITY);
-        xTaskCreatePinnedToCore(tx_task, "ESP32-CAN TX", OPENMRN_STACK_SIZE,
-            this, TX_TASK_PRIORITY, &txTaskHandle_, tskNO_AFFINITY);
+        xTaskCreate(rx_task, "CAN RX",
+            config_arduino_openmrn_stack_size(), this,
```

Comment:
i think these should be different constants. These tasks almost always need a different amount of stack if you wantto be careful about RAM usage.

(feel free to ignore this if you intend to remove these tasks in the future).

Reply:
This class is planned to be removed with the merge of the TWAI, there are a couple other bugs in this implementation that went unnoticed for some time (counting rx/tx frames as an example)